src/main/java/seedu/address/model/person/Star.java

```diff
@@ -0,0 +1,60 @@
+package seedu.address.model.person;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.util.AppUtil.checkArgument;
+
+/**
+ * Represents a Student's stars in the address book.
+ */
+public class Star {
+
+    public static final String MESSAGE_CONSTRAINTS =
+            "Stars given should be more than 0.";
+
+    public static final Star NOSTAR = new Star(0);
+
+    public final Integer noOfStars; // number of stars given to a student
```

Comment:
Maybe numOfStars sounds more comprehensible? noOf might be misleading to some.

Reply:
yep definitely!